sherpa.sh

```diff
@@ -0,0 +1,40 @@
+package: SHERPA
+version: "%(tag_basename)s"
+source: https://github.com/alisw/sherpa
+tag: "alice/v2.1.1"
+build_requires:
+  - autotools
+  - GCC-Toolchain
+---
+#!/bin/bash -e
+
+rsync -a --delete --exclude '**/.git' --delete-excluded $SOURCEDIR/ ./
+
+autoreconf -ivf
+./configure --prefix=$INSTALLROOT \
+            --with-sqlite3=install
```

Comment:
Do you really need sqlite? If yes, we probably should have a separate package for that (eventually picking it up from the system).